loopchain/blockchain/blockchain.py

```diff
@@ -78,8 +75,9 @@ def __init__(self, channel_name=None, store_id=None, block_manager=None):
         self.__peer_id = ChannelProperty().peer_id
         self.__block_manager: BlockManager = block_manager
 
-        store_id = f"{store_id}_{channel_name}"
-        self._blockchain_store, self._blockchain_store_path = utils.init_default_key_value_store(store_id)
+        # FIXME : old_store_id is temporary code to rename dbpath
+        old_store_id = f"{ChannelProperty().peer_target}_{channel_name}"
```

Comment:
Just question
What is the difference between ChannelProperty().peer_id and peer_target?
I just worry about the case when peer_id is not the same as peer_target.